keyboards/primekb/prime_e/keymaps/milestogo/config.h

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2019 Holten Campbell
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x5052
+#define PRODUCT_ID      0x0051
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    PrimeKB
+#define PRODUCT         Prime_E
+#define DESCRIPTION     Ergo_45
```

Comment:
You should need all of this, as it's already there, and doesn't appear to be changed. 

```suggestion
```